### PR TITLE
Update the SegmentedControl to support more use cases.

### PR DESCRIPTION
**Pull request by kofi-clever**
Update the SegmentedControl to support more use cases.

Ran into a few corner cases when using SegmentedControl in context in the
Schools Dashboard.

- Reducing font size to better fit with typical text sizes.
- Polishing up styling issues.
- Adding a disabled state.
- Updating hover state cursors.

seg-ctrl

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "dev-server": "node_modules/.bin/webpack-dev-server --content-base docs/ --inline --watch"
   },
   "dependencies": {
+    "classnames": "^2.2.5",
```
Comment (jacobsimon) [inline package.json]: Do we need this

File: src/SegmentedControl/SegmentedControl.less [hunk 2]
```diff
@@ -1,29 +1,63 @@
+@import "../colors.less";
```
Comment (jacobsimon) [inline src/SegmentedControl/SegmentedControl.less]: 💯

**Comment by jacobsimon** [comment]
Looks great

File: package.json (hunk 1 above)
Comment (kofi-clever) [inline package.json]: been using it for convenience and for cleaning up code related to conditional classes, but we can do without it for now.
worth considering, though. it's relatively small (~1K unminified).

stripped it out, along with some of the CSS colors we're not using here yet - https://github.com/Clever/components/pull/36/commits/06335a20e202543f9504af8c0f4faf6f73376902

**Pull request merged**